modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs

```diff
@@ -227,6 +227,13 @@ private void _ExportBeginImpl(string[] features, bool isDebug, string path, long
                         projectDataDirName = Path.Combine("Contents", "Resources", projectDataDirName);
                     }
 
+                    // Delete old publish output files to prevent problems with file mixing.
+                    string projectDataDir = Path.Join(ProjectSettings.GlobalizePath("res://"), Path.GetDirectoryName(path), projectDataDirName);
```

Comment:
Not sure I trust this to always result in the correct path.

Reply:
In my tests, the 'path' is the relative path to the project's 'res' directory as the root (export_presets.cfg stores the 'export_path' is like that), and I just briefly tested the directory exported to 'res' under Linux and a directory that is not 'res', and maybe more testing is needed.